src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx

```diff
@@ -421,7 +442,21 @@ export const CommentContainer: FunctionComponent<Props> = ({
     comment.lastViewerAction !== "CREATE" &&
     comment.lastViewerAction !== "EDIT";
 
-  return (
+    if(comment.body === 'DELETED'){
+       return (<div
+        style={{
+          border: '2px solid gray',
+          borderRadius: '3px',
+          padding: '8px',
+          backgroundColor: 'lightgray',
+        }}
```

Comment:
Coral uses css styles imported via generated typescript types. Please use those instead of inline styles as that way, the bundled css can be optimized by Webpack!